src/canisters/frontend/ic-certified-assets/src/certification/types/http.rs

```diff
@@ -27,6 +34,7 @@ pub struct HttpResponse {
     pub status_code: u16,
     pub headers: Vec<HeaderField>,
     pub body: RcBytes,
+    pub upgrade: bool,
```

Comment:
Is this worthy of mention in the changelog?

Reply:
I'm not sure, but added it anyways. I consider it a move from implicit to explicit, so not really worthy of calling it out, but it may affect people importing the library.